Assets/SteamVR_Unity_Toolkit/Scripts/Abstractions/VRTK_WorldPointer.cs

```diff
@@ -244,6 +245,10 @@ protected virtual void TogglePointer(bool state)
         {
             var playAreaState = (showPlayAreaCursor ? state : false);
             playAreaCursor.gameObject.SetActive(playAreaState);
+            if (!state && interactableObject && interactableObject.pointerActivatesUseAction && interactableObject.holdButtonToUse)
```

Comment:
should you also check here if the interactable object `IsUsing()` before calling stop using?

If someone has some code in their StopUsing() call that relies on something that was in the StartUsing() and for some reason this object is no longer not using (i dunno how! but people are creative :p)

Then they may not want the stop using to be called (unless it's in the process of being used)